src/hal/lolin32_lora.h

```diff
@@ -1,24 +1,26 @@
-// Hardware related definitions for lolin32_lora shield
+// Hardware related definitions for lolin32 loraNode32 shield
 // See https://github.com/hallard/LoLin32-Lora
 
 #define HAS_DISPLAY U8X8_SSD1306_128X64_NONAME_HW_I2C // OLED-Display on board
-#define HAS_LED GPIO_NUM_25 // white LED on board
-#define HAS_BUTTON 15 // button "PROG" on board
+#define HAS_LED NOT_A_PIN   // Led os on same pin than Lora SS pin, to avoid pb, we don't use it
```

Comment:
Not sure if this works?
Since "NOT_A_PIN" is not evaluated yet in main.cpp in 

> void set_onboard_led(int st){
> #ifdef HAS_LED
>     switch (st) {
>         case 1: digitalWrite(HAS_LED, HIGH); break;
>         case 0: digitalWrite(HAS_LED, LOW); break;
>     }
> #endif
> };